qfieldsync/gui/photo_naming_widget.py

```diff
@@ -52,7 +52,6 @@ def addLayerFields(self, layer_source):
         for i, field in enumerate(layer.fields()):
             row = self.rowCount()
             ews = layer.editorWidgetSetup(i)
-
             if ews.type() == "ExternalResource":
```

Comment:
can we reduce diff and keep the nice separation between vars and ifs?

Reply:
@suricactus , done -- sorry, I pushed a wrong commit earlier, some more things to review.